recipes/utils/mongo-replicaset/readme.md

```diff
@@ -38,6 +38,9 @@ digraph Cluster {
 
 **IMPORTANT:** This recipe is not yet ready for production environments. See _further improvements_ section for more details.
```

Comment:
do we need this still i think the new code is quite good, no?

Reply:
As mentioned in skype, followed your suggestion and removed in ab1d81e